run hack/test-cmd.sh failed

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): BUG REPORT


**What happened**:
```
# ./build/run.sh hack/test-cmd.sh
....
message:error: stat /tmp/newconfig.yaml: no such file or directory
has not:Error loading config file
1853 hack/make-rules/test-cmd.sh
!!! Error in hack/make-rules/test-cmd.sh:223
  'return 1' exited with status 1
Call stack:
  1: hack/make-rules/test-cmd.sh:223 runTests(...)
  2: hack/make-rules/test-cmd.sh:2976 main(...)
Exiting with status 1
+++ [1221 13:33:12] Clean up complete
Makefile:258: recipe for target 'test-cmd' failed
make: *** [test-cmd] Error 1
!!! Error in ./build/../build/common.sh:570
  '"${docker_cmd[@]}" "${cmd[@]}"' exited with status 2
Call stack:
  1: ./build/../build/common.sh:570 kube::build::run_build_command_ex(...)
  2: ./build/../build/common.sh:504 kube::build::run_build_command(...)
  3: ./build/run.sh:30 main(...)
Exiting with status 1

```

--- a/hack/make-rules/test-cmd.sh
+++ b/hack/make-rules/test-cmd.sh
@@ -1849,7 +1849,7 @@ __EOF__
 
   # test invalid config
   kubectl config view | sed -E "s/apiVersion: .*/apiVersion: v-1/g" > "${TMPDIR:-/tmp}"/newconfig.yaml
-  output_message=$(! "${KUBE_OUTPUT_HOSTBIN}/kubectl" get pods --context="" --user="" --kubeconfig=/tmp/newconfig.yaml 2>&1)
+  output_message=$(! "${KUBE_OUTPUT_HOSTBIN}/kubectl" get pods --context="" --user="" --kubeconfig="${TMPDIR:-/tmp}"/newconfig.yaml 2>&1)
   kube::test::if_has_string "${output_message}" "Error loading config file"
 
   output_message=$(! kubectl get pod --kubeconfig=missing-config 2>&1)

